Jenkinsfile

```diff
@@ -48,63 +48,108 @@ pipeline {
         }
       }
     }
-
-    stage("All Test Only") {
-      agent { label 'master || (linux && immutable)' }
-      options { skipDefaultCheckout() }
-      when {
-        beforeAgent true
-        changeRequest()
-      }
-      steps {
-        runJob('All')
-      }
-    }
     /**
       launch integration tests.
     */
     stage("Integration Tests") {
-      agent { label 'master || (linux && immutable)' }
-      options { skipDefaultCheckout() }
-      when {
-        beforeAgent true
-        allOf {
-          anyOf {
-            not {
-              changeRequest()
+      parallel {
```

Comment:
change to use the parallel declarative step, it is a little more verbose but we remove the "All Test Only" stage, now we run the "All" stage instead on PRs and skip the rest, this makes the UI more consistent